Initial mobile design

erickma1 opened the pull request:
Add index.html file 
Add style.css
Created header section
Create headline section
Add images (icons for social media handles)
Add background image

erickma1 committed:
Add linters.yml

Add linters to .github\workflows directory

erickma1 committed:
Revert "Add linters.yml"

This reverts commit 1ec6630ca09a008d6b1ac5c6ced9cbc07bd2bf19.

erickma1 committed:
Add linters.yml

Add linters.yml to .github\workflows\ directory

erickma1 committed:
Add .gitignore

Add .gitignore and add node_modules to the first line

erickma1 committed:
Add .hintrc

erickma1 committed:
Add .stylelintrc.json

erickma1 committed:
Add background image

erickma1 committed:
Add angelist icon

erickma1 committed:
Add git icon

erickma1 committed:
Add status icon

erickma1 committed:
Add linkedin icon

erickma1 committed:
Add logo

erickma1 committed:
Add medium icon

erickma1 committed:
Add time icon

erickma1 committed:
Add twitter icon

erickma1 committed:
Add union icon

erickma1 committed:
Add index file

Add the main page (index.html) this file is the homepage

erickma1 committed:
Update readme file

Add project description

erickma1 committed:
Add style.css

erickma1 committed:
Update README.md

erickma1 committed:
Update style.css

Corrected linter errors in style.css

erickma1 commented:
Resolved errors in style.css

erickma1 committed:
Update Readme.md

usorfaitheloho commented on README.md:
```diff
@@ -1,2 +1,160 @@
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Please remove these lines of code from the license section of your code as it just gives you details about what the license is about 👍

usorfaitheloho commented on index.html:
```diff
@@ -0,0 +1,51 @@
+            </nav>
+        </section>
+        <section class="headline">
+            <div class="headline-title">
+                Hey There<br />
+                I'm Eric.
+            </div>
```
The main title of a website is a `h1` tag for search engine optimization you can check this  [article](https://www.reliablesoft.net/h1-tag/) 😃

usorfaitheloho commented on index.html:
```diff
@@ -0,0 +1,51 @@
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.
+            </p>
+            <ul class="headline-social">
+                <li><img src="/images/icons/twitter.png" alt="twitter"></li>
+                <li><img src="/images/icons/linkedin.png" alt="linkedIn"></li>
+                <li><img src="/images/icons/medium.png" alt="medium"></li>
+                <li><img src="/images/icons/git.png" alt="github"></li>
+                <li><img src="/images/icons/angelist.png" alt="angellist"></li>
+            </ul>
```
You did a great job wrapping your social icons in a `ul` tag, please each icon should be in a `a href` tag` in the `li` tag , like this 

>     <li> <a></a></li>

Also, in the styling, the icons and the text should have spacing similar to that in the design and it shouldn't cluster with the background image. kindly remember to use the right `font type ` as well 😄 

Figma design :
![image](https://user-images.githubusercontent.com/20537712/235202292-0ab87095-e078-40d4-9776-764770252baf.png)

Your design :
![image](https://user-images.githubusercontent.com/20537712/235202427-06e9516a-9e61-43be-8fc3-3385c4dc8e61.png)

usorfaitheloho commented on index.html:
```diff
@@ -0,0 +1,51 @@
+
+
+<body>
+    <main class="top-section">
+        <section class="header">
+            <div class="app-bar">
+                <ul>
+                    <li class="app-bar-item1"><img src="/images/icons/time.png" alt="Time"></li>
+                    <li class="app-bar-item2"><img src="/images/icons/Group4.png" alt="Phone Status"></li>
+                </ul>
+            </div>
+            <nav class="menu-icon">
+                <ul>
+                    <li class="menu-item1"><img src="/images/icons/logo.png" alt="Logo"></li>
+                    <li class="menu-item2"><img src="/images/icons/Union.png" alt="Main menu"></li>
+                </ul>
+            </nav>
+        </section>
```
I'd suggest you make the following changes to this part of your code ;

- [ ] Please the first section in the `body` of your portfolio is the `header` tag,to show the heading of a webpage containing the logo and navigation. That is, `line 15-28` should be moved to start from`line 14` and the `section` tag should be replaced with a `header` tag 😄  

- [ ] The time and battery bars you included is not a part of the main design and should not be included in your portfolio, as it just shows the mobile time and battery status when the design was taken . Therefore, I'd suggest you kindly remove `lines 16 - line 21` from your code 👍 
- [ ] The logo icon on `line 24`   should be in a `a href` tag to show good user semantics and convention

usorfaitheloho reviewed (changes requested):
## Hi @erickma1   :clap:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/rZ10JtoPRv1Ly/giphy.gif)

## To Highlight :clap:  
:arrow_forward: Git/Github flow best practices has been followed :heavy_check_mark: 
:arrow_forward: Good code quality :heavy_check_mark: 
:arrow_forward: Good figma design :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

erickma1 committed:
Add angel.svg

erickma1 committed:
Add disabled angel svg

erickma1 committed:
Add git svg

erickma1 committed:
Add disabled git svg

erickma1 committed:
Add git disabled svg

erickma1 committed:
Add linkedin svg

erickma1 committed:
Add linked disabled svg

erickma1 committed:
Add medium svg

erickma1 committed:
Add disabled medium svg

erickma1 committed:
Add twitter svg

erickma1 committed:
Add disabled twitter svg

erickma1 committed:
Update Readme.md

Removed comments from Readme.md file

erickma1 committed:
Update index.html

Change fonts and add links to social media icons

erickma1 committed:
Update style.css

Add formatting for sections and text

erickma1 commented:
Remove comments from Readme.md 
Added h1 tag
Put social media tags in a href tag
Changed font type and aligned icons properly
Removed battery and status icons
Put logo icon in a href tag

LionRouge1 commented on index.html:
```diff
@@ -0,0 +1,47 @@
+                    <li class="menu-item2"><img src="images/icons/Union.png" alt="Main menu"></li>
+                </ul>
+            </nav>
+        </section>
```
- [ ] As mentioned by the previous review, kindly replace the `<section class="header">` with a `header` tag and move the the logo link out side of the `nav` tag for semantics reasons. Your code could look like this:

```sh
<header class="header">
    <a href="index.html"><img src="images/icons/logo.png" alt="Logo" /></a>
    <nav class="menu-icon">
        <div class="menu-item2"><img src="images/icons/Union.png" alt="Main menu"></div>
    </nav>
</header>
<main class="main_content">
        <section class="headline">
            ...
        </section>
</main>
```

That will make your code clear DRY :muscle:

LionRouge1 reviewed (changes requested):
Hi @erickma1, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost](https://media.giphy.com/media/R2m2NzVxQ3pbG/giphy.gif)

## Suggested changes :recycle: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

erickma1 committed:
Update index.html

Replace class header with header tag and moved logo link outside of nav

erickma1 committed:
Update style.css

erickma1 commented:
Replace class header with header tag and moved logo link outside of nav

zieeco commented on index.html:
```diff
@@ -14,14 +14,14 @@
+        </header>
         <section class="headline">
             <h1 class="headline-title">
                 Hey There<br />
```
**[OPTIONAL BUT REQUIRED]**


- Hey, great job resolving the issue raised by the previous reviewer, however, your code should look like the code snippet given by the previous reviewer. You've wrapped the header inside of the main tag which is not a good practice and a good use of the semantic tag in your page layout. please, kindly remove the header from the tag. see this [link](https://stackoverflow.com/questions/20470345/should-the-header-and-footer-tags-be-inside-the-main-tag) for more details.

NOTE: *Please, fix this before you merge as this would be checked for your next project milestone. It's best if you fixed it now* :pray:

zieeco reviewed (approved):
# status: APPROVED :green_circle: :100: 

Hi, @erickma1, 


<img height="230" src="https://media1.giphy.com/media/2yLNN4wTy7Zr8JSXHB/giphy-downsized-medium.gif"/>


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
